### ISSUE TITLE
Crash in the Great Swamps

### ISSUE THREAD

**Issue by K0stov** [SurDno/Uncut#3]
I was using this mod and the game crashed. Not sure if it's due to the mod.

```
! [LUA][ERROR] ERROR: Path between [mar_smart_terrain_4_7] and [mar_smart_terrain_6_7] doesnt exist for [27].
 
FATAL ERROR
 
[error]Expression    : fatal error
[error]Function      : CScriptEngine::lua_error
[error]File          : E:\priquel\sources\engine\xrServerEntities\script_engine.cpp
[error]Line          : 180
[error]Description   : <no expression>
[error]Arguments     : LUA error: ...\s.t.a.l.k.e.r. clear sky\gamedata\scripts\_g.script:975: bad argument #2 to 'format' (string expected, got nil)
```

**Comment by SurDno** [comment]
Fixed in 4d82a8261ab83ab5101a4628cd08ae0eb2f1a421.

**Comment by SurDno** [comment]
Fixed in 4d82a8261ab83ab5101a4628cd08ae0eb2f1a421.
